### ISSUE TITLE
global tranlastion campaign content setting -> features

### ISSUE THREAD

**Issue by angaither** [DoSomethingArchive/legacy-website#5010]
export campaign translation settings to features
`entity_translation_settings_node__campaign`

on `/admin/config/regional/entity_translation`: 
- [ ] default language should be set to "Author language"
- [ ] hide language selector should be selected
- [ ] exclude language neutral from the available languages should be selected

those should be checked and exported via `dosomething_global` feature
basically it should look like this:

![image](https://cloud.githubusercontent.com/assets/6330971/9615147/623f33b6-50c4-11e5-9c2a-66736139e3e8.png)

refs #4934
